src/rootUtils.h

```diff
@@ -30,6 +30,78 @@ constexpr static auto metaTreeName = "podio_metadata";
  */
 constexpr static auto paramBranchName = "PARAMETERS";
 
+/**
+ * Names of the fields with the keys and values of the generic parameters for
+ * the RNTuples until map types are supported
+ */
+constexpr static auto intKey = "GPIntKeys";
+constexpr static auto floatKey = "GPFloatKeys";
+constexpr static auto doubleKey = "GPDoubleKeys";
+constexpr static auto stringKey = "GPStringKeys";
+
+constexpr static auto intValue = "GPIntValues";
+constexpr static auto floatValue = "GPFloatValues";
+constexpr static auto doubleValue = "GPDoubleValues";
+constexpr static auto stringValue = "GPStringValues";
+
+/**
+ * Get the name of the key depending on the type
+ */
+template <typename T>
+constexpr auto getGPKey() {
```

Comment:
```suggestion
constexpr auto getGPKeyName() {
```
I was slightly confused at first how this would get keys without a connection to the map while reading things at the calling site.

Reply:
Done